src/main/resources/profiles/metaarchive.json

```diff
@@ -56,6 +57,9 @@
    ],
    "Allow-Fetch.txt":false,
    "Serialization":"optional",
+   "Accept-Serialization": [
+      "application/tar"
+   ],
```

Comment:
Should the `Accept-BagIt-Version` below also include `1.0`?

Reply:
Updating the `Accept-BagIt-Version` to include `1.0` makes sense to me. I'll double check the other profiles as well to make sure they accept `1.0`.